### PR TITLE
Do not include the absolute children outline for groups.

### PR DESCRIPTION
**Pull request by seanparsons**
**Problem:**
The absolute children outline for groups matches the outline for the group itself, so if there's any lag in rendering it then it looks a bit strange.

**Fix:**
Exclude group children from the absolute children outline.

**Commit Details:**
- Added `testId` to `OutlineControlProps`.
- `OutlineControl` now includes a `data-testid` value so it can be easily identified.
- `AbsoluteChildrenOutline` filters out selected views which are group like.
- Changed some types from `Array<ElementPath>` to `ReadonlyArray<ElementPath>` to make them more compatible with `arrayAccumulate`.

### CODE REVIEW

**Review by ruggi — APPROVED** [review]
\o\

File: editor/src/components/canvas/controls/select-mode/simple-outline-control.tsx [hunk 1]
```diff
@@ -28,26 +28,32 @@ export const MultiSelectOutlineControl = React.memo<MultiSelectOutlineControlPro
...
-        ...localSelectedElements.map((path) => (
-          <OutlineControl
-            key={EP.toString(path)}
-            targets={[path]}
-            color='primary'
-            outlineStyle='solid'
-          />
-        )),
+        ...localSelectedElements.map((path) => {
+          const varSafeId = `multiselect-element-outline-${EP.toVarSafeComponentId(path)}`
```
Comment (Rheeseyb) [inline editor/src/components/canvas/controls/select-mode/simple-outline-control.tsx:38]: I don't believe there's any need for this to be "varSafe". I actually don't think we need `toVarSafeComponentId` at all anymore, as that's left over from the old old Utopia

**Review by Rheeseyb — APPROVED** [review]

File: editor/src/components/canvas/controls/select-mode/simple-outline-control.tsx (hunk 1 above)
Comment (seanparsons) [reply editor/src/components/canvas/controls/select-mode/simple-outline-control.tsx:38]: ORLY?

Comment (seanparsons) [reply editor/src/components/canvas/controls/select-mode/simple-outline-control.tsx]: Removed in b26e6a405.

Comment (Rheeseyb) [reply editor/src/components/canvas/controls/select-mode/simple-outline-control.tsx]: Yeah, it was used for creating variable names that were valid in JS. I don't recall specifically why.

**Pull request merged**